packages/react-charting/src/components/CommonComponents/CartesianChart.types.ts

```diff
@@ -519,4 +524,15 @@ export interface IModifiedCartesianChartProps extends ICartesianChartProps {
    * Callback method used when mouse leaves the chart boundary.
    */
   onChartMouseLeave?: () => void;
+
+  /**
+   *@default false
+   *Used for to elipse y axis labes and show tooltip on x axis labels
+   */
+  showYAxisLablesTooltip?: boolean;
```

Comment:
are these applicable only for horizontal bar chart with axis. If yes can these be moved to horizontal bar chart with axis component